rice/detail/NativeRegistry.ipp

```diff
@@ -68,7 +68,12 @@ namespace Rice::detail
 
       if (k == klass && m == method_id)
       {
-        return std::any_cast<Return_T>(d);
+        auto* ptr = std::any_cast<Return_T>(&d);
+        if (!ptr)
+        {
+          rb_raise(rb_eRuntimeError, "Unexpected return type for %s %s", rb_class2name(klass), rb_id2name(method_id));
```

Comment:
Tiny nit, but should this be `%s#%s` to match normal Ruby nomenclature? `Unexpected return type for MyClass#method_name`.

Reply:
Yeah, it could. Just updated.